src/sass/accordion/_base.scss

```diff
@@ -4,76 +4,49 @@
 @use 'core' as *;
 
 .#{$prefix}-accordion {
-  &__title {
-    font-size: $ts-16;
+  border-bottom: 1px solid $color-black-100;
+
+  &__summary {
+    border-top: 1px solid $color-black-100;
   }
 
-  &__title button {
+  &__toggle {
     align-items: center;
     background-color: transparent;
     border: none;
-    cursor: pointer;
     display: flex;
-    margin: 0;
-    padding: 0;
+    padding: $spacing-sm;
+    text-align: left;
+    -webkit-appearance: none;
     width: 100%;
-
-    &:focus {
-      box-shadow: 0 0 0 $spacing-xxs/2 $color-white-base, 0 0 0 $spacing-xxs $color-blue-600;
-      outline: none;
-    }
-
-    &:hover {
-      background-color: $color-black-100;
-    }
-
-    svg {
-      margin-right: $spacing-xs;
-    }
-  }
-
-  &__title button[aria-expanded='true'] {
-    svg {
-      transform: rotate(90deg);
-    }
   }
 
-  &__content[hidden] {
+  &__toggle[aria-expanded='true'] &__icon-bar {
     display: none;
   }
 
-  &--panel {
-    border: 1px solid $color-black-200;
-    border-radius: $spacing-xxs;
+  &__toggle:hover {
+    background-color: $color-black-100;
+    cursor: pointer;
   }
 
-  &--panel &__title {
-    background-color: $color-black-000;
-    border-radius: $spacing-xxs;
+  &__toggle:focus {
+    outline: 2px solid $color-blue-base;
+    outline-offset: $spacing-xxs/2;
   }
 
-  &--panel &__title button {
-    border-radius: $spacing-xxs;
-    padding: $spacing-sm*.75;
-
-    &:hover {
-      background-color: $color-black-100;
-    }
+  &__toggle-text {
+    font-size: $ts-18;
+    font-weight: $font-weight-bold;
+    padding-right: $spacing-sm;
   }
 
-  &--panel &__title button[aria-expanded='true'] {
-    border-bottom-left-radius: 0;
-    border-bottom-right-radius: 0;
+  &__toggle-icon {
+    color: $color-black-600;
+    margin-left: auto;
   }
 
-  &--panel &__content {
-    border-top: 1px solid $color-black-200;
-    padding: $spacing-sm;
-  }
-}
-
-.rvt-collapsible-accordion {
-  .rvt-collapsible:not(:first-child) {
-    margin-top: $spacing-sm;
+  &__panel {
+    padding: $spacing-md $spacing-md;
```

Comment:
Wondering if this could be written simply as `padding: $spacing-md;` or if there's a specificity issue that doubling up this way handles.

Reply:
Oh, yeah, good call. I think they were different values at one point, but since they're the same, I'll just use the one value/shorthand.